controllers/placementapi_controller.go

```diff
@@ -66,6 +66,11 @@ import (
 	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
 )
 
+const (
+	//NovaNetworkAttachmentsReadyInitMessage
+	NovaNetworkAttachmentsReadyErrorMessage = "NetworkAttachments error occurred not all pods have interfaces with ips as configured in NetworkAttachments: %s"
```

Comment:
possibly not `Nova` ? see comment bellow, could we just update the one in lib-common?